_posts/2021-07-30-quarkus-2x-platform-quarkiverse-registry.adoc

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: 'Platforms and Streams: a new way to discover Quarkus extensions'
+date: 2021-07-30
+tags: release
+synopsis: 'The platform model and supporting tools available in Quarkus 2.1.0.Final make it easy to discover and work with the Quarkus extension ecosystem.'
+author: aloubyansky
+---
+= Platforms and Streams: a new way to discover Quarkus extensions
+
+As you may have seen from recent Quarkus Insights episodes (#48, #55 and #57, specifically), we've made a number of changes to our tools in the 2.x stream to improve the developer experience.  When 2.0 was released, a few sharp-eyed Quarkus client users spotted some new options and we're excited to explain what those are all about with the release of Quarkus 2.1.
+
+**TL;DR:** We have a whole new set of more granular BOMs that you can use instead of the big `quarkus-universe-bom`. We have deployed a registry service that aligns and manages platform compatibility across these more granular BOMs and our tools (maven, gradle and client) know how to work with this registry to simplify the discovery of extensions that work with your project.
```

Comment:
```suggestion
**TL;DR:** We have a whole new set of more granular BOMs that you can use instead of the big `quarkus-universe-bom`. We have deployed a registry service that aligns and manages platform compatibility across these more granular BOMs and our tools (maven, gradle and CLI) know how to work with this registry to simplify the discovery of extensions that work with your project.
```